evalai/urls.py

```diff
@@ -27,7 +27,7 @@
 handler404 = 'evalai.views.page_not_found'
 handler500 = 'evalai.views.internal_server_error'
 
-urlpatterns = [url(r'^',
+urlpatterns = [url(r'^api/web/',
```

Comment:
Can you add a url pattern for the home view like `url(r'^$', web.views.home, name="home"),` so that we open the `home()` view at `/`